doc/related_work.md

```diff
@@ -0,0 +1,35 @@
+# Related Work
+
+## Standards
+
+[ASAM OpenDrive](https://www.asam.net/standards/detail/opendrive/):
+Road networks are specified in the OpenDrive format.
+It is used to test simulation models in a co-simulation directly in the GitHub CI pipeline.
+
+[ASAM OpenScenario](https://www.asam.net/standards/detail/openscenario/):
+How objects move in a scene is described in the OpenScenario format.
+It is used to test simulation models in a co-simulation directly in the GitHub CI pipeline.
+
+[ASAM OpenSimulationInterface](https://github.com/OpenSimulationInterface/open-simulation-interface):
+All simulation models on OpenMSL use OSI as the standard interface for model input and model output.
+
+[Modelica FMI](https://fmi-standard.org/):
```

Comment:
Is OSMP officially a part of ASAM OSI or should it be mentioned here as well as FMI "template"?

Reply:
I added OSMP to the tools.